src/ListOfContacts/HubSpotListOfContactsClient.cs

```diff
@@ -50,6 +51,63 @@ public HubSpotListOfContactsClient(string apiKey)
               apiKey)
         { }
 
+
+        /// <summary>
+        /// Creates the a new contact list entity asynchronously.
+        /// https://legacydocs.hubspot.com/docs/methods/lists/create_list
+        /// </summary>
+        /// <param name="entity">The entity.</param>
+        /// <returns></returns>
+        /// <exception cref="NotImplementedException"></exception>
+        public async Task<T> CreateAsync<T>(CreateContactListRequestHubSpotEntity payload) where T : IHubSpotEntity, new()
+            //where T : IHubSpotEntity, new()
```

Comment:
**nit-pick**: commented code, please remove it

Reply:
That is not a "nit-pick" @nover - it was sloppy of me. I shall remove the commented code.